ppr-api/src/ppr_api/models/financing_statement.py

```diff
@@ -297,56 +297,34 @@ def save(self):
         db.session.commit()
 
     @classmethod
-    def find_all_by_account_id(cls, account_id: str = None, staff: bool = False):
+    def find_all_by_account_id(cls, account_id):
         """Return a summary list of recent financing statements belonging to an account."""
-        statement_list = None
-        if account_id:
-            # No date range restriction for staff?
-            if staff:
-                statement_list = db.session.query(Registration.registration_ts,
-                                                  Registration.registration_num,
-                                                  Registration.registration_type,
-                                                  FinancingStatement.state_type).\
-                                filter(FinancingStatement.id == Registration.financing_id,
-                                       Registration.account_id == account_id,
-                                       Registration.registration_type_cl.in_(['PPSALIEN', 'MISCLIEN', 'CROWNLIEN'])).\
-                                order_by(FinancingStatement.id).all()
-            else:
-                days_ago = model_utils.now_ts_offset(10, False)
-                statement_list = db.session.query(Registration.registration_ts,
-                                                  Registration.registration_num,
-                                                  Registration.registration_type,
-                                                  FinancingStatement.state_type).\
-                                filter(FinancingStatement.id == Registration.financing_id,
-                                       Registration.account_id == account_id,
-                                       Registration.registration_type_cl.in_(['PPSALIEN', 'MISCLIEN', 'CROWNLIEN']),
-                                       Registration.registration_ts > days_ago).\
-                                order_by(FinancingStatement.id).all()
-
         results_json = []
-        if not statement_list:
+        if not account_id:
             return results_json
-        #    raise BusinessException(
-        #        error=f'No Financing Statements found for Account ID {account_id}.',
-        #        status_code=HTTPStatus.NOT_FOUND
-        #    )
-
-        for statement in statement_list:
-            if staff or statement.state_type == model_utils.STATE_ACTIVE:
-                statement_json = {
-                    'matchType': 'EXACT',
-                    'createDateTime': model_utils.format_ts(statement.registration_ts),
-                    'baseRegistrationNumber': statement.registration_num,
-                    'registrationType': statement.registration_type
+
+        query = model_utils.QUERY_ACCOUNT_FINANCING_STATEMENTS.replace('?', account_id)
+        results = db.session.execute(query)
```

Comment:
I like using binds, for lots of reasons.
```
QUERY_ACCOUNT_FINANCING_STATEMENTS = """
SELECT...
   AND r.account_id = :search_account
...
results = db.session.execute(query, {'search_account': account_id})
```

Reply:
Change made in second commit.